docs/src/content/docs/utilities/Injectors/create-injectable.md

```diff
@@ -0,0 +1,57 @@
+---
+title: createInjectable
+description: A function based approach for creating Injectable services
+entryPoint: create-injectable
+badge: experimental
+contributor: josh-morony
+---
+
+`createInjectable` uses `createInjectionToken` behind the scenes as a way to
+create injectable services and other types of injectables in Angular without
+using classes and decorators.
+
+The general difference is that rather than using a class, we use a `function` to
+create the injectable. Whatever the function returns is what will be the
+consumable public API of the service, everything else will be private.
+
+To create an injectable that is `providedIn: 'root'` you can omit the `isRoot`
+configuration.
+
+## Usage
+
+### Shared Service
+
+```ts
+// defining a shared service
+export const [MyService] = createInjectable(() => {
```

Comment:
Looking at the code, I think that `createInjectable` does not return an Array anymore.